js/display-pictures.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { Description } from './mock.ts';
+
+const pictureList = document.querySelector('.pictures');
+const pictureTemplate = document.querySelector('#picture')?.querySelector('.picture');
+
+const picturesFragment = document.createDocumentFragment();
+
+const displayPictures = (dataArray: Description[]) => {
+	dataArray.forEach(({url, description, likes, comments}) => {
+		const pictureElement = pictureTemplate?.cloneNode(true) as HTMLAnchorElement;
```

Comment:
Если кинули ошибку. Сработал ассершн, и тогда вопрос не нужен
```suggestion
		const pictureElement = pictureTemplate.cloneNode(true) as HTMLAnchorElement;
```
Если ошибку не кинули, то можно для ТС уточнить, что мы уверены что якорь найден
```suggestion
		const pictureElement = pictureTemplate!.cloneNode(true) as HTMLAnchorElement;
```

Reply:
Мне вот еще не совсем понятно, где-то мы можем указывать тип через "as", а где-то через <>. Не портится ли единообразие от этого?

Reply:
А вообще, спасибо большое, Артур, все по фактам раскидал, у меня вопросов не осталось. Только боюсь я тебя так замучаю к концу курса)

Reply:
> Мне вот еще не совсем понятно, где-то мы можем указывать тип через "as", а где-то через <>. Не портится ли единообразие от этого?

`as` это жесткий тайпассерт, а ` <>` мы уточняем дженерик, там где он задокументировать.
`cloneNode` выдает не тот тип, который нам нужен. 

Лучше я на созвоне уточню